docs/v0.10/Readme.md

```diff
@@ -573,6 +606,8 @@ public function categories(){
 
 Now, when you have save the results from your Many-to-Many relationship the ID's of each selected value will be synced and added to your pivot table.
 
+> Please note that this can also be configured as a normal `Select Dropdown`.
```

Comment:
> Please note that this can also be configured as a normal `Select Dropdown`.

Please note that this can also be configured as a normal `Select Dropdown` in case you need a simple multiple selection with fixed `options` instead of `relationship`.